### PR TITLE
Add pygitversion recipe

### PR DESCRIPTION
**Pull request by steven-murray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by steven-murray** [comment]
@conda-forge-admin please ping team

Sorry to bother you, but I'm having trouble getting it to build -- it's saying the package `setuptools_scm` is not found, but it exists on both PyPI and conda-forge, so I'm a bit confused.

**Comment by scopatz** [comment]
It is probably a complaint about the fact that you are building from a tarball, which you have to for conda-forge

**Comment by steven-murray** [comment]
@scopatz I'm sorry but I don't understand -- are you saying that it cannot build from a tarball, but it has to? Does this mean it's impossible?

The error that I'm getting doesn't sound like something complaining about a tarball -- it's complaining that it can't find one of the dependencies (even though it exists on both pypi and conda-forge).

File: recipes/pygitversion/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  build:
```
[the remark this answers is not in the document]
Comment (steven-murray) [reply recipes/pygitversion/meta.yaml]: Hmmm, that doesn't seem to have worked. Which sdist do you think is broken on PyPI? The one for my code (`pygitversion`) or that for `setuptools_scm`?

**Pull request closed without merging**